src/Transport/TransportFactoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Transport;
+
+use Sentry\Options;
+
+interface TransportFactoryInterface
```

Comment:
Please add a docblock with the description of the purpose of this interface, e.g. `This interface defines a contract for all classes willing to create instances of the transport to use with the Sentry client.`